svc/pkg/game-user/ops/recommend/tests/integration.rs

```diff
@@ -1,6 +1,93 @@
 use chirp_worker::prelude::*;
+use proto::backend::{self, pkg::*};
+
+use game_user::create::Response as GameUser;
+
+async fn create_game_users(
+	ctx: &TestCtx,
+	count: usize,
+	namespace_id: Option<Uuid>,
+) -> (Uuid, Vec<GameUser>) {
+	let mut users = vec![];
+
+	let namespace_id = namespace_id.unwrap_or_else(Uuid::new_v4);
+
+	println!("Creating {count} game users...");
+
+	for i in 0..count {
+		println!("Creating game user {i}...");
+
+		let operation = op!([ctx] game_user_create {
+			namespace_id: Some(namespace_id.into()),
+			user_id: Some(Uuid::new_v4().into())
+		});
+
+		users.push(operation.await.expect("Error creating game user!"));
+
+		println!("Game user {i} created.");
```

Comment:
This is all good code, but too verbose, especially for a test. Would rewrite as a simple:

```rust
for i in 0..count {
  tracing::info!(?i, "creating game user");
  
  let user = op!([ctx] game_user_create {
	  namespace_id: Some(namespace_id.into()),
	  user_id: Some(Uuid::new_v4().into())
  }).await.unwrap();
  users.push(user);
}
```

I love the attention to logging, but you could even skip the `creating game user` log, since the operation call will be logged regardless. You could use `tracing::trace!` if you _really_ want that log there.

If it panics, we'll have a stack trace to see where it is with the `unwrap`.

Reply:
> ...since the operation call will be logged regardless.

Good to know! I think I made it extra verbose just to be safe. 😅